addons/binding/org.openhab.binding.unifi/src/main/java/org/openhab/binding/unifi/internal/handler/UniFiClientThingHandler.java

```diff
@@ -195,80 +218,85 @@ private boolean isClientHome(@Nullable UniFiClient client) {
 
     private void refreshChannel(@Nullable UniFiClient client, ChannelUID channelUID) {
         // mgb: only refresh if we're ONLINE
-        if (getThing().getStatus() == ONLINE) {
-            logger.debug("Refreshing channel = {}", channelUID);
+        // if (getThing().getStatus() == ONLINE) {
```

Comment:
Why is this commented out?

Reply:
Changed - Thing status is now used again.